src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -158,162 +208,236 @@ const PointShare = props => {
-            <Row className="teammate-one">
-              <img
-                className="teammate-one-avatar"
-                src={props.team.child1.AvatarURL}
-                alt="Child Avatar"
-              />
-            </Row>
-            <Row className="teammate-two">
-              <img
-                className="teammate-one-avatar"
-                src={props.team.child2.AvatarURL}
-                alt="Child Avatar"
+
+      <div className="point-share-rectangle">
+        <div className="point-share-text">
+          <h2 className="point-share-text-heading">POINTS SHARING</h2>
+          <p className="point-share-paragraph">{`Now it is time to split up 100 points across your squad's portfolio of stories and drawings.  You must put a minimum of 10 points on each and the total among all 4 must add up to 100 points.`}</p>
+
+          <p className="point-share-paragraph">
+            {`As you read both stories and look at both drawings, think about which ones best reflect the characters,  plot, and settings from the chapters you read.  Share the most points with the ones you think are the best.`}{' '}
+          </p>
+        </div>
+      </div>
+
+      <div className="point-share-boxes-container">
+        <div className="point-share-box child-one">
+          <div className="point-share-child-info">
+            <img src={childOneImg} alt="child-one-avatar" />
+            <p className="point-share-heading">{`${user.characterName.toUpperCase()} (YOU)`}</p>
+          </div>
+          <div className="point-share-story">
+            <p className="point-share-heading">STORY</p>
+            <div className="point-share-submission">
+              <img src={sampleWritingSubmission} alt="child-one-story" />
+            </div>
+            <div className="point-share-zoom-container">
+              <ZoomInOutlined
+                className="point-share-zoom"
+                style={{ fontSize: '40px', padding: '5.5px' }}
+                onClick={() => openModal([{ ImgURL: sampleWritingSubmission }])}
               />
-            </Row>
-          </Col>
-          <Col>
-            <Row className="teammate-one-points">
-              <div className="submission-container">
-                <img
-                  className="submission"
-                  src={props.team.child1.ImgURL}
-                  alt="Submission"
-                  onClick={() =>
-                    openModal([{ ImgURL: props.team.child1.ImgURL }])
-                  }
-                />
-                <InputNumber
-                  value={illustrationOnePoints}
-                  max={70}
-                  min={0}
-                  step={5}
-                  onChange={value =>
-                    sharePointHandler({
-                      value,
-                      pointSetter: setIllustrationOnePoints,
+            </div>
+            <div className="counter-box">
+              <p className="points-assigned">{storyOnePoints}</p>
```

Comment:
Needed to use a <p> tag here as opposed to an InputNumber so that we can implement the increment and decrement carats per the Figma.